@here/harp-mapview-decoder/lib/TileLoader.ts

```diff
@@ -255,77 +176,60 @@ export class TileLoader implements ITileLoader {
                     return;
                 }
 
-                this.onDecoded(decodedTile);
+                this.decodedTile = decodedTile;
```

Comment:
you have a private method: `onDecoded` that does the same thing as these two lines

Reply:
done